source/manual/deploying.html.md

```diff
@@ -17,27 +17,44 @@ If you are responding to a security incident, follow the steps to [deploy fixes
 [reg]: https://gds.blog.gov.uk/2012/11/02/regular-releases-reduce-risk
 [sec]: /manual/deploy-fixes-for-a-security-vulnerability.html
 
-## 1. Test on integration
+## Deploying a branch
+
+It can be useful to test your branch in a non-local environment before it is merged to master. Note that you should only do this on Integration, unless you have a good reason to deploy to a different environment.
+
+Announce your deployment in `#govuk-deploy` as a minimum, so that people are aware a non-master branch is being built. If you anticipate your release causing problems, it's worth announcing in `#govuk-2ndline` too.
```

Comment:
I don't think this is necessary all the time.

Reply:
Can you elaborate? Would be good for the docs to provide clear expectations of when an announcement in #govuk-deploy is necessary.

Reply:
I don't think it's necessary to announce integration deploys in #govuk-deploy. Generally I think there is an expectation that integration is running new/different code compared to staging/production so developers should expect it to not necessarily be working the same as production. If there is a reason we need to keep integration on master, it'll often be made clear via an email to developers or in #govuk-2ndline (an example of this is when content designers are doing training).

I imagine there are still times when announcing to #govuk-2ndline is useful (if you anticipate that the deploy will cause problems as you've already written) but I don't think requiring developers to announce in #govuk-deploy all the time is necessary.

Reply:
The paragraph below 

> Announce your deployment in `#govuk-2ndline` if it’s potentially problematic, or in `#govuk-deploy` if it involves deploying something other than a release tag (like a branch).

which has been removed in this commit, I think refers specifically to deploying a branch to staging/production and in that case I think it is useful to announce in #govuk-deploy.

Reply:
Cool, think that can be summed up by:

> If you are deploying to staging or production, announce your deployment in `#govuk-deploy` so that people are aware a non-master branch is being built. It is not necessary to announce if you're deploying to Integration.

Have now pushed up a new commit 👍